src/components/Log/LogItemEdit.vue

```diff
@@ -39,9 +39,11 @@ export default {
   },
   setup() {
     const route = useRoute();
-    const { id } = route.params;
-    const { query } = route.query;
-    return { id, query };
+    const { itemId } = route.params;
+
+    return {
```

Comment:
If the object just contains `itemId`, does it need to be returned in a list like this?

Reply:
you could write itemId = route.params.itemId, but i find the other way cleaner as it just destructures the itemId out of the parent route.params object

Reply:
Also, this is the way it's written in the Ionic docs. You could write it the other way with no issue, but it seems like a convention thing.